packages/server/utils/analytics/analytics.ts

```diff
@@ -94,38 +89,31 @@ class Analytics {
     template: MeetingTemplate
   ) => {
     meetingMembers.forEach((meetingMember) =>
-      this.meetingEnd(
-        meetingMember.userId,
-        completedMeeting,
-        meetingMembers,
-        createMeetingTemplateAnalyticsParams(template)
-      )
+      this.meetingEnd(meetingMember.userId, completedMeeting, meetingMembers, template)
     )
   }
 
   private meetingEnd = (
     userId: string,
     completedMeeting: Meeting,
     meetingMembers: MeetingMember[],
+    template?: MeetingTemplate,
     meetingSpecificProperties?: any
   ) => {
-    const {facilitatorUserId, meetingNumber, meetingType, phases, teamId} = completedMeeting
-    const presentMemberUserIds = meetingMembers.map(({userId}) => userId)
-    const wasFacilitator = userId === facilitatorUserId
     this.track(userId, 'Meeting Completed', {
-      hasIcebreaker: phases[0]?.phaseType === 'checkin',
-      // include wasFacilitator as a flag to handle 1 per meeting
-      wasFacilitator,
-      userIds: wasFacilitator ? presentMemberUserIds : undefined,
-      meetingType,
-      meetingNumber,
```

Comment:
+1 are we intentionally losing `meetingNumber` here?

Reply:
Yes! It feels to me the actual meetingNumber, like, "this is retro #86", does not help much quite in the meeting level of events.